spec/ParseUser.spec.js

```diff
@@ -1265,6 +1265,36 @@ describe('Parse.User testing', () => {
     done();
   });
 
+  fit('log in with Facebook and save signed up User with verifyUserEmails=true and preventLoginWithUnverifiedEmail=true', async done => {
+    const emailAdapter = {
+      sendPasswordResetEmail: () => Promise.resolve(),
+      sendMail: () => Promise.resolve(),
+    };
+    await reconfigureServer({
+      appName: 'unused',
+      verifyUserEmails: true,
+      preventLoginWithUnverifiedEmail: true,
+      emailAdapter: {
+        module: emailAdapter,
+      },
+      publicServerURL: 'http://localhost:8378/1',
+    });
+
+    const provider = getMockFacebookProvider();
+    Parse.User._registerAuthenticationProvider(provider);
+    const user = await Parse.User._logInWith('facebook');
+    expect(user instanceof Parse.User).toBeTrue();
+    expect(Parse.User.current()).toEqual(user);
+    expect(user.extended()).toBeTrue();
+    expect(provider.authData.id).toBe(provider.synchronizedUserId);
+    expect(provider.authData.access_token).toBe(provider.synchronizedAuthToken);
+    expect(provider.authData.expiration_date).toBe(provider.synchronizedExpiration);
+    expect(user._isLinked('facebook')).toBeTrue();
+
+    await user.save();
+    done();
```

Comment:
Done is not needed

Reply:
I had added `done() ` because i got JASMINE timeout error without it. Using expectSync doesn't cause this issue so now pushing test with this change

Reply:
Removing `done()` result in   `error  'done' is defined but never used  no-unused-vars` when i try to commit.

Reply:
Figured out from another test. Learning JS on the go with your help :). hank you.